deploy/README.md

```diff
@@ -0,0 +1,16 @@
+# Deploy
+
+## Deploy to labs.arxiv.org
+
+### Build the image
+1. cd arxiv-docs
+1. export TAG=  ...Find a commit tag in git for the docker image tag.
+1. docker build . -t arxiv/labs:$TAG -f ./Dockerfile-labs
+1. docker push arxiv/labs:$TAG
+1. docker run -it -p 8000:8000 arxiv/labs:$TAG
+
+### Deploy the image
+1. ssh arxiv-bastion
+1. helm ls | grep labs-static
```

Comment:
Just curious, what is the purpose of this line?

Reply:
Just for developer sanity. Ahead of changes, helm-ls shows me the current state: that it's running, current revision, and last update date.

Reply:
Thanks.